lib/asana/http_client/error_handling.rb

```diff
@@ -40,12 +40,18 @@ def handle(num_timeouts=0, &request)
           when 404 then raise not_found(e.response)
           when 412 then recover_response(e.response)
           when 429 then raise rate_limit_enforced(e.response)
-          when 500 then raise server_error(e.response)
```

Comment:
in fact, this was never called since 500s are treated as `Faraday::ServerError`

Reply:
Thanks for catching 👍